tests/components/husqvarna_automower/test_switch.py

```diff
@@ -153,6 +154,61 @@ async def test_stay_out_zone_switch_commands(
     assert len(mocked_method.mock_calls) == 2
 
 
+@pytest.mark.parametrize(
+    ("service", "boolean", "excepted_state"),
+    [
+        ("turn_off", False, "off"),
+        ("turn_on", True, "on"),
+        ("toggle", True, "on"),
```

Comment:
```suggestion
        (SERVICE_TURN_OFF, False, STATE_OFF),
        ("turn_on", True, "on"),
        ("toggle", True, "on"),
```
would be nice if the rest also gets constants